src/app/components/Drbb/DrbbContainer.jsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import axios from 'axios';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router';
+
+import appConfig from '../../data/appConfig';
+import DrbbItem from './DrbbItem';
+import { createResearchNowQuery } from '../../utils/researchNowUtils';
+
+class DrbbContainer extends React.Component {
+  constructor(props, context) {
+    super();
+    this.search = context.router.location.search || '';
+    this.query = context.router.location.query || {};
+    this.state = {
+      works: [],
+      drbbResultsLoading: true,
+    };
+  }
+
+  componentDidMount() {
+    this.fetchResearchNowResults();
+  }
+
+  fetchResearchNowResults() {
+    axios(`${appConfig.baseUrl}/api/research-now${this.search}`)
+      .then((resp) => {
+        if (!resp.data || !resp.data.works) {
+          this.setState({ drbbResultsLoading: false });
+          return;
+        }
+        this.setState({
+          works: resp.data.works,
+          totalWorks: resp.data.totalWorks,
+          drbbResultsLoading: false,
+        });
+      })
+      .catch(console.error);
+  }
+
+  content() {
+    const {
+      works,
+      drbbResultsLoading,
+      totalWorks,
+    } = this.state;
+
+    if (drbbResultsLoading) {
+      return (
+        <div className="drr-loading-layer">
+          <span
+            className="loading-animation loadingLayer-texts-loadingWord"
+          >
+            Loading results
+          </span>
+          <div className="loadingDots">
+            <span />
+            <span />
+            <span />
+            <span />
+          </div>
+        </div>
+      );
+    }
+
+    if (works && works.length) {
+      console.log(createResearchNowQuery(this.query));
+      return ([
```

Comment:
Why is this wrapped in an array?

Reply:
Because there are two elements. I could also use `React.Fragment`

Reply:
Huhh I've always used fragment instead of array, but if it works it works!